platform/jewel/samples/ide-plugin/src/main/kotlin/org/jetbrains/jewel/samples/ideplugin/SwingComparisonTabPanel.kt

```diff
@@ -197,7 +196,7 @@ internal class SwingComparisonTabPanel : BorderLayoutPanel() {
     }
 
     private fun Panel.comboBoxesRow() {
-        row("ComboBoxes:") {
+        row("Combo Boxes:") {
```

Comment:
Not sure why the change (they're always called "combobox", not "combo box", elsewhere in Jewel)

Reply:
To align with the official JB documentation https://plugins.jetbrains.com/docs/intellij/combo-box.html#0